apps/idos-enclave/src/pages/App.tsx

```diff
@@ -102,11 +102,11 @@ export function App({ store, enclave }: AppProps) {
       throw new Error(`Unexpected request from parent: ${requestData.intent}`);
 
     responsePort.current = ports[0];
+    setEncryptionUserPublicKey(event.data.message.expectedUserEncryptionPublicKey);
```

Comment:
```suggestion
    setEncryptionUserPublicKey(requestData.message?.expectedUserEncryptionPublicKey);
```
This seems unsafe just looking at the rest of the file, wdyt of this alt?

Reply:
Also, how does `setEncryptionUserPublicKey` handle empty/null/undefined values?

Reply:
we can either throw an error "No Public encryption key found" OR just log and error with same message. what do u think?

Reply:
Throw. Let's make it loudly blow up.